src/components/App.tsx

```diff
@@ -53,11 +60,24 @@ export default class App extends React.Component<undefined, AppState> {
     }
 
     public render(): React.ReactElement<{}> {
+        let manifestUrl = "https://readium2.herokuapp.com/pub/L2FwcC9jaGlsZHJlbnMtbGl0ZXJhdHVyZS5lcHVi/manifest.json";
         return (
```

Comment:
Just out of interest: have you managed to integrate `r2-streamer-js` in the Electron app (as a NPM package). Depending on which Electron version you are using at the moment (see latest https://github.com/electron/electron/blob/master/.node-version ) you will be able to pick a particular build target above ES6-2015, see https://unpkg.com/r2-streamer-js/dist/